src/Business Foundation/App/NoSeries/src/Upgrade/NoSeriesInstaller.Codeunit.al

```diff
@@ -0,0 +1,37 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Foundation.NoSeries;
+
+using System.Upgrade;
+
+codeunit 329 "No. Series Installer"
+{
+    Subtype = Install;
```

Comment:
This seems to do the logic of an upgrade codeunit. Is there a reason it's an install codeunit instead?

Reply:
Should be both

Reply:
As Darrick says this should be on both, hence keeping it here.